doc/source/serve/autoscaling-guide.md

```diff
@@ -22,29 +22,40 @@ deployments:
 
 ## Autoscaling Basic Configuration
 
-Instead of setting a fixed number of replicas for a deployment and manually updating it, you can configure a deployment to autoscale based on incoming traffic. The Serve autoscaler reacts to traffic spikes by monitoring queue sizes and making scaling decisions to add or remove replicas. Configure it by setting the [autoscaling_config](../serve/api/doc/ray.serve.config.AutoscalingConfig.rst) field in deployment options. The following is a quick guide on how to configure autoscaling.
+Instead of setting a fixed number of replicas for a deployment and manually updating it, you can configure a deployment to autoscale based on incoming traffic. The Serve autoscaler reacts to traffic spikes by monitoring queue sizes and making scaling decisions to add or remove replicas. Turn on autoscaling for a deployment by setting `num_replicas=auto`. You can further configure it by tuning the [autoscaling_config](../serve/api/doc/ray.serve.config.AutoscalingConfig.rst) in deployment options.
```

Comment:
Non-blocker: Maybe we can do `num_replicas="auto"` instead so the user know this is a literal "auto" and not an enum that they need to be imported from somewhere (when using it on the deployment decorator). Also would be nice to have an example of the actual deployment decorator using it.

Same goes to other places doing `num_replicas=auto`.

Reply:
Done! The example of an actual deployment decorator is there already, it's part of the code block from a python file :D
<img width="589" alt="Screenshot 2024-03-15 at 12 12 49 PM" src="https://github.com/ray-project/ray/assets/15851518/627cb8b0-31fd-43f5-b451-e5758366a35a">